core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3105,4 +3281,182 @@ public Long getValue() {
 
     private Metrics() {} // prevent instantiation
   }
+
+  /**
+   * An audit context for file system master.
+   */
+  @NotThreadSafe
+  public final class MasterAuditContext implements AuditContext {
```

Comment:
Done. And I've also renamed it to `FileSystemMasterAuditContext` because this class is really just about file system master auditing.